roles/rciam-metrics/defaults/main.yml

```diff
@@ -11,6 +11,7 @@ metrics_api_url: http://localhost:8004/
 metrics_hostname: "metrics-dev.rciam.grnet.gr"
 metrics_protocol: "http"
 metrics_api_path: "/api/v1"
+fastapi_log_level: "info"
```

Comment:
Do we expect log level configuration for other components of the metrics service? Is there a reason not go with a technology-neutral approach, e.g.:
```
metrics_api_log_level: "info"
metrics_api_syslog_identifier: "metrics_server_fastapi"
```

or to group log related variables:
```
metrics_api_logging:
  level: "info"
  syslog_identifier: "metrics_server_fastapi"
```